stat.go

```diff
@@ -934,18 +929,19 @@ func MeanVariance(x, weights []float64) (mean, variance float64) {
 			compensation += d
 		}
 		variance = (ss - compensation*compensation/float64(len(x))) / float64(len(x)-1)
-		return
```

Comment:
continue != returning early. You're really testing for (a != 0 || b != 0) to do the operation. Testing for the opposite and skipping the operation is not desirable. There is also no reduction in nesting since the if statement has to be there.